README.md

```diff
@@ -39,7 +39,9 @@ You should also specify settings that will be shared across all the plugin rules
       "createClass": "createReactClass", // Regex for Component Factory to use,
                                          // default to "createReactClass"
       "pragma": "React",  // Pragma to use, default to "React"
-      "version": "15.0", // React version, default to the latest React stable release
+      "version": "detect", // React version, "detect" automatically picks the version from your project's package.json.
```

Comment:
This is technically not entirely correct though. `package.json` (generally) defines a range, whereas "detect" will detect the exact version you have installed.

Reply:
yes, you are right, so do we actually check package lock?

Reply:
I'll defer the final wording to @ljharb though.

Reply:
okay that makes sense.